src/cf-buttons/usage.md

```diff
@@ -361,6 +362,26 @@ For accessibility reasons, use the semantic `<button>` instead of a link when po
 <input type="submit" value="Input Tag" class="a-btn a-btn__super active">
 ```
 
+### Full-width buttons on x-small screens
+
+_Reduce screen size to see these in action_
```

Comment:
I wonder if we could use container queries for purposes of examples in documentation, which presumably will always be viewed on a browser that supports them.